src/rpc/rawtransaction.cpp

```diff
@@ -39,6 +41,7 @@
 #include <script/sign.h>
 #include <script/signingprovider.h>
 #include <script/standard.h>
+#include <sequentia/rpc/rawtransaction.h>
```

Comment:
The same as I mentioned in `createrawtransactions`. Put it in a distinct block of includes so we try as much to aviod conflicts in diffs.